pkg/server/server.go

```diff
@@ -52,15 +52,22 @@ func getScanningFilterFromConfigMap(filePath string) ([]string, error) {
 func readConfigScanningFilters() (skipList []string, err error) {
 	configDir := `/config`
 	configSkipListPath := filepath.Join(configDir, `skiplist`)
+	datreeSkipConfigDir := filepath.Join(configDir, `datreeSkipList`)
```

Comment:
suggest rename: `datreeSkipListConfigDir`

Reply:
Maybe I am wrong but the new file is  inside `datreeSkipList` no? does this code works?

Reply:
something is wired here, why we use `filepath.Join` twice?